portfolio/src/main/webapp/contacts.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>LadyPython contacts</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,700;0,900;1,100;1,200;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <a href="index.html">About me</a>
+    </header>
+    <div class="content">
+      <ul id="contacts">
+        <li id="phone">+79259181314</li>
+        <li id="mail">sofia.ladypython@gmail.com</li>
+        <li id="telegram">@LadyPython</li>
+      </ul>
+      <!-- <p>or write me right now:</p>
+      <form id="contact" action="mailto:sofia.ladypython@gmail.com" method="post">
+        <textarea type="text" placeholder="Hire me..." onfocus="this.placeholder = ''" onblur="this.placeholder = 'Hire me...'"></textarea>
+        <button type="submit" form="contact" value="Send" onclick="sendForm('contact-form-container')">Send</button>
+      </form>
+      <div id="contact-form-container"></div> -->
```

Comment:
Please either fill out or remove for this commit (i.e., do not leave commented parts in the code).